bears/elm/ElmLintBear.py

```diff
@@ -0,0 +1,37 @@
+from shutil import which
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='elm-format',
+        use_stdin=True,
```

Comment:
@jayvdb why should we avoid stdin? If we use stdin things will be kept in memory and there's less IO right? It also means we can use it for things that don't exist on file systems making the bear easier to reuse in our efforts to get languages in languages analyzed or testing.